js-packages/@prestojs/rest/src/dedupeInFlightRequestsMiddleware.ts

```diff
@@ -0,0 +1,73 @@
+import { EndpointRequestInit, MiddlewareFunction } from './Endpoint';
+
+function defaultGetKey(url: string, requestInit: EndpointRequestInit): string {
+    return [
+        url,
```

Comment:
If I have the same request with query params in a different order then it will be considered distinct. I can easily see the same query being constructed in 2 different places having params in a different order

I don't see any harm in also checking whether the query params match (order should only matter for the same item, ie
- `/foo?p=1&p=2` != `/foo?p=2&p=1`
- `/foo?a=1&b=2` = `/foo?b=2&a=1`

Reply:
yep good one, bad oversight by me